src/docs/cookbook-events/like-feature/index.md

```diff
@@ -0,0 +1,536 @@
+
+First off, we'll need to change what event type we want to filter. In our case, it's `user/updated`.
+
+###### Step 1: Filter events by `user/updated`
+```diff
+const queryEvents = (args) => {
+-  var filter = {eventTypes: "listing/created,listing/updated"};
++  var filter = {eventTypes: "user/updated"};
+  return integrationSdk.events.query(
+    {...args, ...filter}
+  );
+};
+
+```
+
+###### Step 2: Add a function that updates the 'likes' value:
+Next, we'll add a function that updates the 'likes' value in listing extended data by calling the Integration API listings/update endpoint:
+```js
+  /** 
+  * @param {string} listingId
+  * @param {number} likeAddition – A value representing a like or a dislike that is either added to or subtracted from currentLikes
+  */
+  const updateListing = (listingId, likeAddition) => {
+    return integrationSdk.listings.query({
+      ids: listingId,
+    })
+    .then(listings => {
+      const listing = listings.data.data[0];
+      const currentLikes = listing.attributes.publicData.likes || 0;
+      const updatedLikes = currentLikes + likeAddition;
+      return integrationSdk.listings.update({
+        id: listingId,
+        publicData: {
+            likes: updatedLikes,
+        },
+      }, { expand: true })
+  })
+};
+```
+
+###### Step 3: Add helper functions:
+
+We'll no longer need the `analyzeEvent` or `isPublished` functions found in the boilerplate code. Instead, let's add a few new functions. These functions will help us determine if the event we've received is a like or dislike, and reduce multiple likes to a single API call:
+
+```js
+// Get the difference between two arrays
+const getDifference = (arr1, arr2) => {
+  return arr1.filter(x => !arr2.includes(x))
+}
+
+// Compare the amount of likes in the previous event to the current one to
+// determine which listing was liked or disliked
+const getLikedListingId = (previousLikes, currentLikes) => {
+  if (previousLikes === null) return currentLikes;
+  if (currentLikes === null) return previousLikes;
+  else return previousLikes.length < currentLikes.length ? getDifference(currentLikes, previousLikes) : getDifference(previousLikes, currentLikes);
+}
+
+const getLikeCount = (previousLikes, currentLikes) => {
+  return previousLikes === null || previousLikes.length < currentLikes.length ? 1 : -1;
+}
+
+// Reducer returns an object with listing ID's as keys and amount of likes as values
+const sortEvents = events => {
```

Comment:
Instead of `sort` in the fn name, the action here could be `group`, as the likes are grouped for each listing? So `groupEvents` or even `groupLikes`.